tools/alpine/Makefile

```diff
@@ -9,25 +9,25 @@ default: push
 show-tag:
 	@sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
-hash: Dockerfile Makefile packages
+iid: Dockerfile Makefile packages
```

Comment:
I think you should still delete the `iid` file after `tag` and `push` as we always need to rebuild in case the outside world has changed, it doesnt look like it will now.